scripts/check-docs.sh

```diff
@@ -1,8 +1,9 @@
 #!/usr/bin/env bash
 
 set -uo pipefail
-
-if [[ $(git diff --stat docs/commands | wc -l) != 0 ]]; then
-  echo "./docs/commands has changes"
+commands=$(git diff --stat docs/commands | wc -l)
```

Comment:
Can we revert this? Why this changes

Reply:
actually much better implementation.
Lets just try keep prs focused for future.v